packages/i18n/data/core.json

```diff
@@ -1,11 +1,4 @@
 {
-  "UIKit.CalendarHeader.nextMonthTooltip": "Next month",
-  "UIKit.CalendarHeader.nextYearTooltip": "Next year",
-  "UIKit.CalendarHeader.previousMonthTooltip": "Previous month",
-  "UIKit.CalendarHeader.previousYearTooltip": "Previous year",
-  "UIKit.CalendarHeader.todayTooltip": "Today",
-  "UIKit.ClearButton.clearButtonLabel": "Clear",
-  "UIKit.CreatableSelectInput.createLabel": "Create \"{inputValue}\"",
```

Comment:
What are these changes for? 🤔

Reply:
It's not part of my change but came when I pulled the latest translations.

Reply:
What do you mean pulled? Which command did you run to get this?

Anyway, please revert these changes as they are not relevant with this PR. Thanks